mozilla/activity-stream
# Bug 1565043 - New TextPromo Component

Testing:

- Change `browser.newtabpage.activity-stream.discoverystream.config` to `{"api_key_pref":"extensions.pocket.oAuthConsumerKey","collapsible":true,"enabled":true,"show_spocs":true,"hardcoded_layout":false,"personalized":false,"layout_endpoint":"https://gist.githubusercontent.com/gvn/0d988c266468ab17c3eeeeab1bed9bef/raw/7976030636ef193878a97f1359ca8c21ba355d4c/text-promo.json"} `
- Change `browser.newtabpage.activity-stream.discoverystream.endpoints` to `https://getpocket.cdn.mozilla.net/,https://spocs.getpocket.com/,file:///,https://gist.githubusercontent.com`
- Confirm layout looks like this:

<img width="878" alt="Screen Shot 2019-08-13 at 1 31 57 PM" src="https://user-images.githubusercontent.com/206379/62975214-c2c34300-bdce-11e9-9a2f-fce12810034f.png">
